setup.py

```diff
@@ -60,6 +61,7 @@ def run(self):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5'
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6'
```

Comment:
Trailing comma? -> cleaner diffs down the road

Reply:
Sure, I could add that. (I just noted it works with 3.7 as well.) I hadn't done that because it didn't seem to be the original author's style.